playwright-e2e/pages/singular/enrollment/consent-form-page.ts

```diff
@@ -47,20 +47,6 @@ export default class ConsentFormPage extends SingularPage {
     return new Input(this.page, { ddpTestID: 'answer:CONSENT_SELF_SIGNATURE_SUBJECT' });
   }
 
-  /**
-   * <br> Question: Your Date of Birth
-   * <br> Type: Input
-   * @param month
-   * @param date
-   * @param year
-   */
-  async dateOfBirth(month: number | string, date: number | string, year: number | string): Promise<void> {
-    const dob = new Question(this.page, { prompt: 'Date of Birth' });
-    await dob.date().locator('input[data-placeholder="MM"]').fill(month.toString());
-    await dob.date().locator('input[data-placeholder="DD"]').fill(date.toString());
-    await dob.date().locator('input[data-placeholder="YYYY"]').fill(year.toString());
-  }
```

Comment:
Moved to `page-base.ts` because similar fields exists in many studies during enrollment workflow.